training/src/main/scala/com/airbnb/aerosolve/training/photon/ml/data/ConvertExamplesToPhotonMLAvroJob.scala

```diff
@@ -0,0 +1,170 @@
+package com.airbnb.aerosolve.training.photon.ml.data
+
+import java.io.File
+
+import com.airbnb.aerosolve.core.{Example, FeatureVector}
+import com.airbnb.aerosolve.core.transforms.Transformer
+import com.airbnb.aerosolve.training.pipeline.GenericPipeline
+import com.typesafe.config.{Config, ConfigFactory, ConfigParseOptions, ConfigResolveOptions}
+import org.apache.avro.generic.{GenericData, GenericRecord}
+import org.apache.avro.mapred.AvroKey
+import org.apache.avro.mapreduce.{AvroJob, AvroKeyOutputFormat}
+import org.apache.avro.Schema
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.Path
+import org.apache.hadoop.io.NullWritable
+import org.apache.hadoop.mapreduce.Job
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.rdd.RDD
+import org.slf4j.LoggerFactory
+
+import scala.collection.JavaConverters._
+import scala.collection.immutable
+import scala.util.Try
+
+
+/**
+  * Providing a data conversion job casting Thrift [[com.airbnb.aerosolve.core.Example]] to PhotonML
+  * Avro format: https://github.com/linkedin/photon-ml#input-data-format
+  *
+  */
+object ConvertExamplesToPhotonMLAvroJob {
```

Comment:
add a template conf file and use the generic pipeline to load it and update the generic pipeline doc, it is easier for others to follow.